packages/react-components/src/components/resource-explorers/explorers/asset-property-explorer/internal-asset-property-explorer.tsx

```diff
@@ -26,10 +26,12 @@ import {
 import { AssetPropertyResource } from '../../types/resources';
 import {
   DEFAULT_ASSET_PROPERTY_TABLE_DEFINITION,
-  DEFAULT_ASSET_PROPERTY_WITH_LATEST_VALUES_TABLE_DEFINITION,
+  createDefaultLatestValuesTableDefinition,
 } from '../../constants/table-resource-definitions';
```

Comment:
would think of moving the file outside of the constants folder since it has functions now

Reply:
makes sense. Ill do it for the decimal places change that follows, bc it uses the same function :)